mitodl/open-discussions
# Display author headline near name on post cards, limit length of headline text

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #677 

#### What's this PR do?
- Displays the post author's headline (if any) near their name on compact and expanded post cards.
- Limits the headline length to 60 characters

#### How should this be manually tested?
- View a channel page (list of compact posts) and a post detail page (expanded post) for a post you authored.  If your profile has a headline it should be visible next to your name. 
- Edit your profile.  You should not be able to enter more than 60 characters for your headline.  Enter a headline if you don't have one or clear it out if you do.
- After you save and change your profile headline, view the channel and post detail pages again.  The changes to your headline should be reflected in the post card(s).

#### Where should the reviewer start?
https://zpl.io/2j4rxL4

#### Screenshots
<img width="812" alt="screen shot 2018-08-07 at 12 25 20 pm" src="https://user-images.githubusercontent.com/187676/43789130-3a437672-9a3d-11e8-86cf-b2def6aa7f1a.png">

<img width="379" alt="screen shot 2018-08-07 at 12 25 38 pm" src="https://user-images.githubusercontent.com/187676/43789132-3c0cd598-9a3d-11e8-997b-3c58e814d82b.png">

<img width="982" alt="screen shot 2018-08-07 at 12 26 05 pm" src="https://user-images.githubusercontent.com/187676/43789141-42ef4436-9a3d-11e8-85ea-4b58f79bfe37.png">
